include/exadg/incompressible_navier_stokes/preconditioners/multigrid_preconditioner_projection.cpp

```diff
@@ -101,6 +101,13 @@ MultigridPreconditionerProjection<dim, Number>::update()
   VectorTypeMG velocity_fine_level = *velocity_multigrid_type_ptr;
   VectorTypeMG velocity_coarse_level;
 
+  // // USELESS?? WHY IS THIS HERE? ##+
+  // VectorTypeMG velocity_coarse_level;
+  // VectorTypeMG velocity_fine_level;
+  // this->get_operator(this->get_number_of_levels() -
+  // 1)->initialize_dof_vector(velocity_fine_level); velocity_fine_level =
+  // *velocity_multigrid_type_ptr;
+
```

Comment:
TODO, I need to test and see if this is needed, I ported this from some other code that had this change